### PR TITLE
Dynamic UI

### PR DESCRIPTION
**Pull request by mariopil**

### CODE REVIEW

File: app/src/Wizard.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            className="modal show"
+            style={{ display: 'block', position: 'initial' }}
+          >
+            <Modal show={showInvokeModal} onHide={handleInvokeModalClose}>
+              <Modal.Header closeButton>
+                <Modal.Title>{modalTitle}</Modal.Title>
+              </Modal.Header>
+              <Modal.Body>
+                <p>
+                  <pre>
```
Comment (eloylp) [inline app/src/Wizard.tsx]: As a nitpick, would it be possible to add a more "terminal-like" style to the invoke modal ? Currently is in this way:

image

By adding this style property  (playing around):

```html
<pre style={{ backgroundColor: 'black' }}>
``` 
I was able to get this :) :

image

I like it more let me know what do you think. Also, i am wondering if the typical button of "copy to clipboard" would be easy to add. 

Thanks for the great work :heart_eyes:

**Review by eloylp — APPROVED** [review]
Great job ! Things that i checked:

* Tried the UI locally. UI works with both contracts Lottery and Voting, correctly replacing all values in the CLI command template when pressing the `invoke` button.
* Code in general, i think is good for the demo.
* That the targets for the first demo are met (10% of the first milestone)
* Just missing the information for the readme but maybe this is for later.

File: app/src/Wizard.tsx (hunk 1 above)
Comment (mariopil) [reply app/src/Wizard.tsx]: I can change it. As for the button - it is there and it works :)

Comment (eloylp) [reply app/src/Wizard.tsx]: Ah yeah, you mean this one right ? for copying the contract code:

image

I am suggesting to add the same, but in the invoke modal. So the user can copy the CLI invokation also :)

Comment (mariopil) [reply app/src/Wizard.tsx]: Ah ok :)

Comment (eloylp) [reply app/src/Wizard.tsx]: Well, maybe needs a bit more of elaboration as, with the code i [shared before](https://github.com/eigerco/nebula/pull/14#discussion_r1262368862), affects every modal ofc .....

image

Comment (mariopil) [reply app/src/Wizard.tsx]: Added both the background and button.

**Review by geofmureithi — APPROVED** [review]
LGTM

File: app/src/Wizard.tsx (hunk 1 above)
Comment (mariopil) [reply app/src/Wizard.tsx]: Great, thanks! :)

**Pull request merged**